Nodejs/Product/Nodejs/SharedProject/ProjectNode.cs

```diff
@@ -1881,7 +1881,7 @@ public virtual void MovePropertyToProjectFile(string propertyName)
         {
             this.Site.GetUIThread().MustBeCalledFromUIThread();
 
-            var prop = this.userBuildProject.GetProperty(propertyName);
+            var prop = this.userBuildProject?.GetProperty(propertyName);
```

Comment:
Is the access on line 1900 safe?

Reply:
Yes, because of this call ` this.EnsureUserProjectFile();`